src/java.base/share/classes/java/lang/reflect/Executable.java

```diff
@@ -55,6 +55,11 @@
     @SuppressWarnings("deprecation")
     Executable() {}
 
+    @SuppressWarnings({"rawtypes"})
+    static final TypeVariable[] NO_TYPE_VARS = new TypeVariable[0];
```

Comment:
Since you asked me offline about some startup noise from this patch I took a look. This line might be another small contributor, since it means we're loading `jlr.TypeVariable` on JVM bootstrap. It would be nice if we could move this to `TypeVariable`, but we don't support private/package-private interface fields for some reason. Shouldn't we though, now that we have private interface fields, etc..?

Reply:
IIRC, Java language-level private interface fields are on the to-do list, but are not a high priority.